neuraxle/base.py

```diff
@@ -2795,12 +3142,11 @@ class Identity(NonTransformableMixin, NonFittableMixin, BaseStep):
     def __init__(self, savers=None, name=None):
         if savers is None:
             savers = [JoblibStepSaver()]
+        BaseTransformer.__init__(self, name=name, savers=savers)
         NonTransformableMixin.__init__(self)
-        NonFittableMixin.__init__(self)
-        BaseStep.__init__(self, name=name, savers=savers)
 
 
-class TransformHandlerOnlyMixin(NonFittableMixin):
+class TransformHandlerOnlyMixin:
```

Comment:
This might not be needed anymore. 

Note: the NonFittableFIxin may still be useful. Not sure if it'll be useful hereto inherit fmor it, but someone who has a BaseStep May still want to remove the fit after.... ideas.

Reply:
it will be deprecated if we judge it as deprecated when doing the DeprecatedMixin issue : https://github.com/Neuraxio/Neuraxle/issues/338